practice_06.08.2019_System.IO/Task1_Folders/Folder.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+
+namespace practice_06._08._2019_System.IO.Task1_Folders
+{
+    public class Folder
```

Comment:
I'd change class name to more applicable. It is not folder, it is like FolderHelper.